index.js

```diff
@@ -37,6 +52,7 @@ module.exports = function serialize(obj, options) {
         options = {space: options};
     }
 
+    var uid = options.uid || UID;
```

Comment:
I don't think it's a good idea to allow `options.uid`. But curious why you added it?